app/models/form526_submission.rb

```diff
@@ -466,8 +467,86 @@ def format_creation_time_for_mailers
     created_at.strftime('%B %-d, %Y %-l:%M %P %Z').sub(/([ap])m/, '\1.m.')
   end
 
+  # Synchronous access to lighthouse API validation boolean for 526 submission
+  # Since this method hits an external API, there could be
+  # exceptions generated for non-200 response codes.
+  #
+  # If return is false then the errors are expected
+  # to be in the lighthouse_validation_errors Array
+  # of Hash.
+  #
+  # NOTE: This is a synchronous access to an external
+  #       API so should not be used within a request/response
+  #       workflow.
+  #
+  def form_content_valid?
+    transform_service = EVSS::DisabilityCompensationForm::Form526ToLighthouseTransform.new
+    body = transform_service.transform(form['form526'])
+
+    begin
+      @lighthoust_validation_response = lighthouse_service.validate526(body)
+    rescue => e
+      fake_lighthouse_response(code: '609', error: e)
```

Comment:
Passing '609' here is redundant since it's already the default. Also, 609 is an interesting response code here. 🤔

Reply:
That would be error 601. 😉